### ISSUE TITLE
MacOS > VoiceOver > VO keys navigation in menu 

### ISSUE THREAD

**Issue by kolaps33** [microsoft/fluentui#14135]
This ticket as as output of the user testing. 

Affected example: Toolbar can contain a menu
URL: https://fluentsite.z22.web.core.windows.net/components/toolbar/definition

Toolbar can contain a menu 
Restriction of the VO cursor movement within the opened menu 

Description: The opened „More“ menu should behave as a group in which the VoiceOver cursor is restricted within, that means the VO + Right or Left arrow keys should not let the user leave the menu. The only way to leave the menu should be possible by pressing the Escape key. Currently, pressing VO + Right or Left arrow after the menu is opened, lets the user to navigate outside of the menu to the toolbar items, which should not be allowed. 
Severity: medium. 

Output of investigation:
Expected: 
In MacOS menu behaves like: 
- VO keys + Space opens menu  
- VO keys + up/down arrow keys navigate between he menu items 
- VO keys + left/right arrow keys don’t make any action 

 
In our example  
In Chrome: 
- VO keys + up/down arrow key don’t navigate between the menu items 
- VO keys + left/right arrow closing the menu 

In Safari: 
- VO keys + up/down arrow key works 
- VO keys + left/right keys escaping the menu  and closing 

**Behaving menu in chrome should be solve in different ticket, high probably report to the chromium, because it behaves differently from Safari. 
Only question here is if we want to try investigate how to prevent closing the menu with using VO keys + left/right arrow keys**
